arch/arm64/boot/dts/qcom/sdm660-xiaomi-lavender-tianma.dts

```diff
@@ -0,0 +1,59 @@
+// SPDX-License-Identifier: GPL-2.0-only
+/*
+ * Copyright (c) 2024, Barnabas Czeman <trabarni@gmail.com>
+ */
+
+/dts-v1/;
+
+#include "sdm660-xiaomi-lavender-common.dtsi"
+
+/ {
+	model = "Xiaomi Redmi Note 7 (Tianma)";
+	compatible = "xiaomi,lavender-tianma", "qcom,sdm660";
```

Comment:
```suggestion
	compatible = "xiaomi,lavender", "qcom,sdm660";
```

Reply:
Or maybe document both now compatibles in qcom.yaml.. if that's what is used upstream for beryllium